cmd/gnoland/main.go

```diff
@@ -42,19 +35,81 @@ var flags struct {
 	rootDir               string
 }
 
-func runMain(args []string) error {
-	fs := flag.NewFlagSet("gnoland", flag.ExitOnError)
-	fs.BoolVar(&flags.skipFailingGenesisTxs, "skip-failing-genesis-txs", false, "don't panic when replaying invalid genesis txs")
-	fs.BoolVar(&flags.skipStart, "skip-start", false, "quit after initialization, don't start the node")
-	fs.StringVar(&flags.genesisBalancesFile, "genesis-balances-file", "./gnoland/genesis/genesis_balances.txt", "initial distribution file")
-	fs.StringVar(&flags.genesisTxsFile, "genesis-txs-file", "./gnoland/genesis/genesis_txs.txt", "initial txs to replay")
-	fs.StringVar(&flags.chainID, "chainid", "dev", "chainid")
-	fs.StringVar(&flags.rootDir, "root-dir", "testdir", "directory for config and data")
-	fs.StringVar(&flags.genesisRemote, "genesis-remote", "localhost:26657", "replacement for '%%REMOTE%%' in genesis")
-	fs.Parse(args)
+func main() {
+	cfg := &gnolandCfg{}
+
+	cmd := commands.NewCommand(
+		"gnoland",
+		commands.Metadata{
+			Name:       "",
```

Comment:
what's the goal?

Reply:
This `Name` param is part of `ffcli`'s Name field, and it's used to discern the actual command being run.
The `Name` is left empty here (and in the root `gnotxport` command) because the binary only expected flags in the past. 

For example, if we fill in the name here with `gnoland`, the command would need to be run, from the binary like this:
`./build/gnoland gnoland [flags]`

Whereas, if we leave this blank, we can run the command with flags as before (how it works now on the repo):
`./build/gnoland [flags]`

I've removed this explicit blank string setting for the `Name` for `gnoland` and the root `gnotxport` command:
[2d1127e](https://github.com/gnolang/gno/pull/497/commits/2d1127e2823b6d6827e88cf814b4772f52da5dbb)